libs/doc-components/src/components/docTokenTable/docTokenTable.css

```diff
@@ -18,9 +20,19 @@
 			background-color: #ffffff !important;
 		}
 
+		.variable {
+			background-color: var(--pine-color-accent);
+			border-radius: var(--pine-dimension-125);
+			color: var(--pine-color-text-primary);
+			padding: var(--pine-dimension-2xs) var(--pine-dimension-xs);
```

Comment:
Just a general comment, and wait for others to chime in. However, I’m not a fan of the purple chips, they make reading the core token colors really odd mentally when they're all purple, but the color naming is not.

Reply:
Purple background color feels a little out of place. Could we just remove the background color?

Reply:
Yep no problem at all